pkg/sqlcmd/sqlcmd.go

```diff
@@ -283,8 +283,13 @@ func (s *Sqlcmd) IncludeFile(path string, processAll bool) error {
 		return err
 	}
 	defer f.Close()
+	info, _ := f.Stat()
+	var maxSize int
 	b := s.batch.batchline
 	scanner := bufio.NewScanner(f)
+	maxSize = int(info.Size())
+	buffer := make([]byte, 0, maxSize)

```

Comment:
The buffer is meant to be the size of the largest token; using the file size seems excessive. Isn't allocation of a huge buffer like this more likely to fail on machines under stress? Or is your issue that the query is a single huge line?

Reply:
yeah, i already changed such logic. As now i'm set it to 2MB size. 
Also, i  have found an issue with using a sql script on macos/linux inside docker. I'm still working on the fix and will push updates as i will finish it.